### PR TITLE
Add functional tests for Mission page

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@davehunt r?

File: tests/functional/test_mission.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+def test_play_video(base_url, selenium):
+    page = MissionPage(base_url, selenium).open()
+    page.play_video()
+    assert not page.is_video_overlay_displayed
+    assert page.is_video_displayed
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_newsletter_default_valuesd(base_url, selenium):
```
Comment (davehunt) [inline tests/functional/test_mission.py]: Typo in `valuesd`

**Comment by davehunt** [comment]
Just a typo to fix, otherwise r+

**Comment by alexgibson** [comment]
Thanks @davehunt!

**Pull request merged**